lib/characters.ts

```diff
@@ -25,3 +26,7 @@ export type Character = {
   charms: Array<string>;
   hide?: boolean;
 };
+
+// not a huge fan of this, but it is simple and easy
+export const PhraestoId = "33";
+export const PhraestoCloneId = "34";
```

Comment:
Personally, I wouldn't do the clone as a different ID. I'd just do 33, and then whenever you need to get an Id, you just always do Math.Floor(id)

Reply:
the clone has a separate tile image (red overlay), which is why i have the two characters as their own unique entries. it made the code side of things much easier than doing a conditional on the image itself

Reply:
I guess that makes sense. I just checked always on the images if it were Phraesto, deal with it then. But yeah, still think this should be a number regardless